### PR TITLE
inductor: support more conv+unary fusion

### PR DESCRIPTION
**Pull request by blzheng**
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* __->__ #92518

cc @mlazos @soumith @voznesenskym @yanboliang @penguinwu @anijain2305 @EikanWang @jgong5 @Guobing-Chen @chunyuan-w @XiaobingSuper @zhuhaozhe @Xia-Weiwen @wenzhe-nrv @jiayisunx @peterbell10 @desertfire

### CODE REVIEW

File: torch/_inductor/mkldnn.py [hunk 1]
```diff
@@ -524,8 +526,21 @@ def create_unary_module(node: torch.fx.node):
         F.gelu: nn.GELU,
         F.relu6: nn.ReLU6,
         F.silu: nn.SiLU,
+        torch.relu: nn.ReLU,
+        torch.sigmoid: nn.Sigmoid,
+        torch.tanh: nn.Tanh,
+        torch.Tensor.relu: nn.ReLU,
+        torch.Tensor.sigmoid: nn.Sigmoid,
+        torch.Tensor.tanh: nn.Tanh,
```
Comment (jgong5) [inline torch/_inductor/mkldnn.py:534]: can we use "relu", "sigmoid" and "tahn" directly as keys for tensor methods?

Comment (blzheng) [reply torch/_inductor/mkldnn.py]: Good point. Changed.

**Review by jgong5 — APPROVED** [review]

**Review by jansel — APPROVED** [review]
